app/views/form/_venue_create_form.erb

```diff
@@ -1,149 +1,300 @@
-<div class="content-form">
+<div class="form-style-10">
+  <h1>Create Venue Form!<span> Do so much more with Burpple </span></h1>
   <%= form_for @venue,url: deals_path, :html => {:multipart => true} do |f| %>
-      <div class = "row">
-        <div class="col-lg-4">
-          <p class="form-text clearfix">
-            <%= f.label "Name of Restaurant*" %>
-            <span>
-              <%= f.text_field :name %>
-            </span>
-          </p>
-        </div>
-      </div>
-
-      <div class="b3 content-subtitle" id="location">Location</div>
-      <div class="content-separator"></div>
 
       <div class = "row">
-        <div class="col-lg-4">
-          <p class="form-text clearfix">
-            <%= f.label "Street Address*" %>
-            <span>
-              <%= f.text_field :street %>
-            </span>
-          </p>
+        <div class="col">
```

Comment:
Seems like you are replacing the default Rails form helper class with your CSS class - will this cause trouble later on? Are we restricted to using the new CSS classes for forms now?